vpr/src/noc/noc_storage.h

```diff
@@ -170,14 +170,14 @@ class NocStorage {
      */
     int layer_num_grid_locs;
 
-    // prevent "copying" of this object
-    NocStorage(const NocStorage&) = delete;
-    void operator=(const NocStorage&) = delete;
-
   public:
     // default constructor (clear all the elements in the vectors)
     NocStorage();
 
+    // prevent "copying" of this object
+    NocStorage(const NocStorage&) = delete;
+    void operator=(const NocStorage&) = delete;
+
```

Comment:
I made the delated constructor and operator public so that when they are used, we get a deleted constructor error message instead of an inaccessible constructor message.